tweepy/api.py

```diff
@@ -211,6 +212,60 @@ def media_upload(self, filename, *args, **kwargs):
             upload_api=True
         )(*args, **kwargs)
 
+    def video_upload(self, filename, *args, **kwargs):
+        """ :reference https://dev.twitter.com/rest/reference/post/media/upload-chunked
```

Comment:
I think this line should be: https://developer.twitter.com/en/docs/media/upload-media/uploading-media/chunked-media-upload